api/apps/api/test/project-planning-areas-tiles/project-tiles.e2e-spec.ts

```diff
@@ -0,0 +1,61 @@
+import { PromiseType } from 'utility-types';
+import { INestApplication, Logger } from '@nestjs/common';
+
+import { bootstrapApplication } from '../utils/api-application';
+import { createWorld } from './world';
+
+let app: INestApplication;
+let world: PromiseType<ReturnType<typeof createWorld>>;
+
+beforeAll(async () => {
+  app = await bootstrapApplication();
+  world = await createWorld(app);
+});
+
+afterAll(async () => {
+  await world.cleanup();
+  await app.close();
+});
+
+describe('PlanningUnitsTilesModule (e2e)', () => {
+
+  test('When creating a regular project whe should be able to access to its planning area tiles', async () => {
+
+    const tile: Buffer = await  world.WhenRequestingTileForProjectPlanningArea(world.regularProjectId);
```

Comment:
nothing to do here, but if you wanted to stick to Arrange/Act/Assert, instead of creating the project in the world setup we could instead return as part of the world `GIvenARegularProjectWasCreated()` function and explicitly set the project up here, then pass its id to `WhenRequestingTileForProjectPlanningArea()`.

We'd need to clean up several projects though, one per test (overhead should be ok, but it will also slow things down a bit).

Reply:
I did try this approach and it was not working (the creation of the project for some reason was not being inserted in the DB)